### DISCUSSION TITLE
Add Tips in Splash screen

### DISCUSSION THREAD

**Discussion by Variable-ind** [Orama-Interactive/Pixelorama#694] [Ideas]
The problem with hint_tooltip is

1. It takes time to show. so users rarely see it and most shortcuts remain *un-discovered*
2. They are limited to buttons only so shortcuts that don't have buttons like these goes un-used for example:
- Alternate tool modes (Ctrl)
- quick color picker (`Alt + Mouse')
- Selection Shortcuts (move without losing image data `Ctrl + Alt + Mouse`)
- and more...
So i suggest a tips dialog in the splash screen. This is what it looks like (Replace the `SplashDialog.tscn` with this one):
[SplashDialog.tscn.zip](https://github.com/Orama-Interactive/Pixelorama/files/8645400/SplashDialog.tscn.zip)

https://user-images.githubusercontent.com/77773850/167259365-2e8ddadb-b295-4d3a-938f-e28f914bc212.mp4

This Discussion is to figure out what tips to put in it and where should the tips be moved to if the current location is not good

**Comment by mrtripie** [comment]
Some more ideas for tips:
- That you can edit the canvas zoom and rotation at the top
- Dragging guides out of rulers
- Greyscale View
- Layout customization
- Pixel Perfect
- Custom brushes
- There's probably timline options that may be good to place here

Also, maybe ones to point out new features on a release (these tips may be temporary)

**Comment by Variable-ind** [comment]
Here's the Repo for splash tip : https://github.com/Variable-ind/Pixelorama/tree/Splash-tips
And executable can be downloaded for testing [HERE](https://nightly.link/Orama-Interactive/Pixelorama/actions/runs/2286923626)
